src/style.css

```
@@ -1 +1,30 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  list-style: none;
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
```

```suggestion
  height: 16px;
  width: 16px;
  margin-right: 4px;
  background-image: url(/src/images/star.svg);
  background-position: 50%;
  background-repeat: no-repeat;
```

Order of your styles is important. Here is useful link - https://9elements.com/css-rule-order/